### PR TITLE
CNV#39982: Cloning VMs assembly

### PR DESCRIPTION
**Pull request by ousleyp**
Version(s): 4.14+

Issue: [CNV-39982](https://issues.redhat.com//browse/CNV-39982)

Link to docs preview:
- https://73715--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/virtual_machines/creating_vms_custom/virt-cloning-vms

QE review:
- [x] QE has approved this change.

Additional information: My main intention for this PR is to create a straightforward "Cloning VMs" assembly to house the cloning-related modules that I am moving out of the instance types documentation.

I considered adding the "About cloning" module to this new assembly, but it might not be relevant unless we add docs for a more technical audience to this page (CLI-based cloning procedures?).

### CODE REVIEW

File: virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-instance-types.adoc
Comment (snarayan-redhat) [inline virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-instance-types.adoc:14]: We need to use a [tech preview snippet](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#indicating-technology-preview-features) as per guidelines.

```suggestion
Creating a VM from an instance type in {VirtProductName} 4.15 and higher is supported for use on {product-title} clusters. In {VirtProductName} 4.14, creating a VM from an instance type is a Technology Preview feature, and is not supported for use on {product-title} clusters.
```

File: virt/virtual_machines/creating_vms_custom/virt-cloning-vms.adoc
Comment (snarayan-redhat) [inline virt/virtual_machines/creating_vms_custom/virt-cloning-vms.adoc:3]: Up to you. Feel free to ignore this comment if not relevant.
```suggestion
= Cloning virtual machines
```

**Review by snarayan-redhat — COMMENTED** [review]
One comment and a suggestion, rest LGTM!
You might also want to consider getting it QE reviewed.

File: virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-instance-types.adoc
Comment (ousleyp) [reply virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-instance-types.adoc:14]: Good catch on that capitalization, thanks! I don't think the TP snippet is necessary here though. My understanding is that because the ROSA docs are versionless, this note is more to say that you should check your OCP Virt version before using instance types. If your CNV version is 4.15, you're good (it's fully supported) but if you are using 4.14 _with ROSA_, instance types are *not* supported because tech preview features are not supported in ROSA. Is that accurate, @mburke5678 ?

**Comment by ousleyp** [comment]
> One comment and a suggestion, rest LGTM! You might also want to consider getting it QE reviewed.

Thanks so much, @snarayan-redhat ! I'm waiting on QE and won't merge until I have QE approval. :)

File: virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-instance-types.adoc
Comment (mburke5678) [reply virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-instance-types.adoc]: @ousleyp @snarayan-redhat Correct. Tech preview features are not supported in ROSA (or OSD).

**Review by jpeimer — APPROVED** [review]
Looks good to me, thank you Pan

**Pull request merged**
